src/main/java/mate/academy/util/ConnectionUtil.java

```diff
@@ -0,0 +1,30 @@
+package mate.academy.util;
+
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+import java.util.Properties;
+
+public class ConnectionUtil {
+    private static final String DB_URL;
+    private static final String DB_USER;
+    private static final String DB_PASSWORD;
+
+    static {
+        Properties properties = new Properties();
+        try (FileInputStream input = new FileInputStream("config.properties")) {
```

Comment:
Ensure that the `config.properties` file is available in the correct location and contains the necessary properties (`db.url`, `db.user`, `db.password`). Otherwise, the application will throw a runtime exception when trying to load the file.